src/rgw/rgw_rados.cc

```diff
@@ -8213,6 +8209,22 @@ int RGWRados::cls_obj_usage_log_read(const string& oid, const string& user, cons
   return r;
 }
 
+int RGWRados::cls_rgw_usage_log_trim_repeat(rgw_rados_ref ref, const string& user, const string& bucket, uint64_t start_epoch, uint64_t end_epoch)
```

Comment:
it doesn't look like this needs to be in RGWRados. try making it static:
`static int cls_rgw_usage_log_trim_repeat(...`
(and removing the declaration from rgw_rados.h)